docs/DEVELOPMENT.md

```diff
@@ -48,32 +48,49 @@ After making your changes, go in the deltachat/electron Dev-console and press `F
 
 ### Code Style <a id="code-style"></a>
 
-We use a combination of [`ESLint`](https://eslint.org) and [`Prettier`](https://prettier.io/) to cover linting/formatting of different types of files.
+#### Checking Code Style
 
-- [`ESLint`](https://eslint.org) is used with [`TypeScript`](https://typescriptlang.org/) rules to catch common bad practices in all `.js`, `.ts` and `.tsx` files
-- [`Prettier`](https://prettier.io/) with rules inspired by [`StandardJS`](https://standardjs.com/) to cover formatting in all `.scss`, `.js`, `.ts`, `.tsx`, `.json` and `.md` files
-
-Commands:
+The primary command for checking any changes made to the code is:
 
 ```sh
-# Check formatting and code problems
-npm run lint
-# Fix autofixable problems and fix formatting
-npm run lint:fix
+npm run check
 ```
 
-Ignore a line:
+This command in turn splits up into the following commands:
```

Comment:
If you want to know more of what's going on, the lower level commands can also be used separately. Once you get the hang of it and get more experienced with the project.